### PR TITLE
feat: add update user

### PR DESCRIPTION
**Pull request by dindalolyta23**

### CODE REVIEW

**Review by niemoo — COMMENTED** [review]
nextnya package-lock.json dan package.json jangan diganti-ganti.

File: src/services/user.js [hunk 1]
```diff
@@ -89,8 +89,37 @@ export async function createUser(payload) {
   return user;
 }
 
+/**
```
Comment (niemoo) [inline src/services/user.js:92]: bisa pakai zod validation type

**Review by niemoo — CHANGES_REQUESTED** [review]

File: src/services/user.js [hunk 2]
```diff
@@ -89,8 +89,37 @@ export async function createUser(payload) {
...
+ *
+ * @param {string} id
+ * @param {Object} payload
+ * @param {string} [payload.name]
+ * @param {string} [payload.email]
+ * @param {string} [payload.phoneNumber]
+ * @param {string} [payload.image]
+ * @param {string} [payload.password]
+ * @returns {Promise<Object>}
+ */
```
Comment (niemoo) [inline src/services/user.js]: bisa disederhanakan pakai zod validation type

File: package.json [hunk 3]
```diff
@@ -55,10 +55,12 @@
     "pino": "^9.5.0",
     "pino-http": "^10.3.0",
     "pino-pretty": "^11.3.0",
+    "plane-ticket-be": "file:",
```
Comment (niemoo) [inline package.json]: sama seperti pr sebelumnya, ada anomali di package-lock.json dan package.json

**Review by niemoo — CHANGES_REQUESTED** [review]

**Comment by ccrsxx** [comment]
Already implemented on 1c9a6011f9c75880aed83fb88f601c0948b4c691

**Pull request closed without merging**